### PR TITLE
Swap: update to be compatible with ERC-20s

### PR DESCRIPTION
**Pull request by fadeev**
To be compatible with the update source code: https://github.com/zeta-chain/example-contracts/pull/85

### CODE REVIEW

**Comment by fadeev** [comment]
@andresaiello please, review. Thanks!

File: docs/developers/omnichain/tutorials/swap.md [hunk 1]
```diff
@@ -74,60 +74,51 @@ contract Swap is zContract {
...
         uint256 outputAmount = SwapHelperLib._doSwap(
             systemContract.wZetaContractAddress(),
             systemContract.uniswapv2FactoryAddress(),
             systemContract.uniswapv2Router02Address(),
             zrc20,
             amount,
-            targetZRC20,
-            minAmountOut
+            targetTokenAddress,
+            0
```
Comment (andresaiello) [inline docs/developers/omnichain/tutorials/swap.md:99]: why did you remove the minOut?

Comment (fadeev) [reply docs/developers/omnichain/tutorials/swap.md:99]: We weren't using it in the tutorial (it was just always 0) as it's not crucial to the core functionality.

**Comment by fadeev** [comment]
Updated to be compatible with https://github.com/zeta-chain/example-contracts/pull/85

**Comment by fadeev** [comment]
@andresaiello please, review.

**Review by andresaiello — APPROVED** [review]
